Lib/typing.py

```diff
@@ -151,7 +151,7 @@ def _type_convert(arg, module=None):
     return arg
 
 
-def _type_check(arg, msg, is_argument=True, module=None):
+def _type_check(arg, msg, is_argument=True, is_class=False, module=None):
```

Comment:
This is not a public API. It starts with an underscore and is undocumented. That being said, it *is* used by Python libraries externally... so while I'm going back and forth on this one, I think that it would be nicest to the users to introduce `is_class=` here just like in the public `ForwardRef`. Otherwise, the backwards compatibility breakage makes it a controversial change to backport as a bugfix.

```suggestion
def _type_check(arg, msg, is_argument=True, module=None, *, is_class=False):
```

Reply:
Thanks a lot, @ambv! I understand the reason behind `is_argument=True, *, is_class=False`, but it still looks kinda inconsistent 😬

Do we need a separate issue to make it both kw-only some day?

Reply:
Yeah, making both kwonly is a separate issue with some complications. `_type_check` we can just change in Python 3.11 without any further ado. `ForwardRef()` will need a deprecation period when the users is calling `is_argument=` and `module=` as non-keyword arguments. But yeah, separate issue.

Reply:
Done! https://bugs.python.org/issue45281

Thanks again!